styles/base.css

```diff
@@ -7,3 +7,18 @@
 body {
   background-color: none;
 }
+
```

Comment:
according to the following code - as i have understood going with bem would mean to give also `details` and `summary` a class name to address them